config/locales/account/en.yml

```diff
@@ -63,6 +63,9 @@ en:
       webauthn_add: Add security key
       webauthn_confirm_delete: Yes, remove key
       webauthn_delete: Remove key
+      webauthn_platform: Platform authenticator
+      webauthn_platform_add: Add platform authenticator
+      webauthn_platform_delete: Remove platform authenticator
```

Comment:
Noting that the confirmation screen still uses "security key" verbiage:

![image](https://user-images.githubusercontent.com/1779930/145406955-cf23e4c7-3331-4295-b206-84fda95c6ff3.png)

Reply:
Yep, there's a good amount of content/translation work to do